tensorflow/python/ops/image_ops.py

```diff
@@ -1004,6 +1007,44 @@ def adjust_contrast(images, contrast_factor):
     return convert_image_dtype(adjusted, orig_dtype, saturate=True)
 
 
+def adjust_gamma(image, gamma=1, gain=1):
+  """Performs Gamma Correction on the input image.
+    Also known as Power Law Transform. This function transforms the 
+    input image pixelwise according to the equation Out = In**gamma 
+    after scaling each pixel to the range 0 to 1.
+
+  Args:
+    image : A tensor.
```

Comment:
I'm afraid not. I passed (shape=(8,8)) images in image_ops_test.py and it works.